ludwig/schema/decoders/utils.py

```diff
@@ -36,6 +38,36 @@ def get_decoder_classes(feature: str):
     return decoder_config_registry[feature]
 
 
+@DeveloperAPI
+def get_decoder_descriptions(feature_type: str):
+    """This function returns a dictionary of decoder descriptions available at the type selection.
+
+    The process works as follows - 1) Get a dictionary of valid decoders from the decoder config registry,
+    but inverse the key/value pairs since we need to index `valid_decoders` later with an altered version
+    of the decoder config class name. 2) Loop through Decoder Metadata entries, if a metadata entry has a
+    decoder name that matches a valid decoder, add the description metadata to the output dictionary.
+
+    :param feature_type: The feature type to get decoder descriptions for
+    :return: A dictionary of decoder descriptions
+    """
+    output = {}
+
+    # Get valid decoder dictionary with following structure:
+    #       key - name of decoder config class altered to match metadata class names,
+    #       value - registered name of decoder
+    valid_decoders = {
+        class_name.__name__.replace("Config", ""): registered_name
```

Comment:
I'm a but confused here. Are the class values of the decoder/encoder/combiner registry have `Config` in the name?

Reply:
Perhaps if we get rid of the `.replace()`, then this approach seems alright. :)

Reply:
Using `module_name` to access the corresponding metadata name now.